.github/workflows/ci.yaml

```
@@ -120,16 +120,32 @@ jobs:
           else
             echo "UNIT_TESTS_IGNORE_PATH=\"server/\"" >> $GITHUB_ENV
           fi
-
+    - name: Set RUN_COVERAGE environment variable
+      run: |
+        if [[ "${{ matrix.python-version }}" == "3.9" ]]; then
+          echo "RUN_COVERAGE=true" >> $GITHUB_ENV
+        else
+          echo "RUN_COVERAGE=false" >> $GITHUB_ENV
+        fi
```

Will this work? a bit shorter
```suggestion
    - name: Set RUN_COVERAGE environment variable
      if: matrix.python-version == '3.9'
      run: echo "RUN_COVERAGE=true" >> $GITHUB_ENV
```